indico/legacy/pdfinterface/latex_templates/inc/contribution.tex

```diff
@@ -180,16 +180,18 @@
         \setlength{\leftskip}{0pt}
     \JINJA{endif}
 
-    \JINJA{for field_value in contrib.field_values}
-        \textbf{\VAR{ field_value.contribution_field.title}}:
+    \JINJA{for field in contrib.event.contribution_fields if field.is_active and field.is_public}
+        \textbf{\VAR{field.title}}:
 
         %% Markdown content
         \rmfamily {
             \allsectionsfont{\rmfamily}
             \sectionfont{\normalsize\rmfamily}
             \subsectionfont{\small\rmfamily}
             \small
-            \VAR{field_value.friendly_data|markdown}
+            \VAR{contrib.get_field_value(field.id) | markdown}
```

Comment:
`get_field_value` iterates over `field_values` internally, so this is much less efficient than the old code